client/src/Components/Townbox/index.tsx

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable react/prop-types */
+import React from 'react'
+import './index.css'
+import {Button} from '../Button'
+
+export interface TownboxProps {
+  /**
+   * Header colour: enter either bg-orange or bg-primary
+   */
+  headerColor?: string
+
+  /**
+   * Heading contents
+   */
+  headerText?: string
+
+  /**
+   * Textbox contents
+   */
+  captionText?: string
+
+  /**
+   * Show visited button?
+   */
+  showButton?: boolean
+
+}
+
+/**
+ * Primary UI component for user interaction
+ */
+export const Townbox: React.FC<TownboxProps> = ({
+  headerText,
+  captionText,
+  headerColor,
+  showButton
+}) => {
+  return (
+    <div className = "w-96 flex">
+      
+      <h2 className = {['font-serif','text-white', 'p-2', 'border-black', 'border-4', 'rounded-md', 'ml-10', 'mt-4', 'absolute', 'z-10', headerColor].join(
+        ' '
+      )}> {headerText} </h2>
+      
+      <div className = "captionboxborder h-52 w-96 absolute mt-12"></div> 
+
+      <div className = "captionbox place-self-center p-2 mt-12 border-black flex relative flex-col items-center justify-center w-full h-52">
+        <p className = "w-4/5 font-sans border-solid pb-2 mb-0 z-10 text-base relative mt-4 break-words">
```

Comment:
Using `w-4/5` might cause some interesting results if we start having variable width boxes. Is padding not enough? you can specify padding only for the x axis with `px-10`